tests/databrowser.js

```diff
@@ -200,6 +204,77 @@ define([
                 util.assertXmlEqual(call("createXML"), PRELOADED_HASHTAGS_XML, {normalize_xmlns: true});
             });
 
+            it("should add the casedb instance when referencing a case in a label", function(done) {
+                util.loadXML("");
+                var mug = util.addQuestion("Text", "mug"),
+                    label = $("[name=itext-en-label]"),
+                    sessionUri = CASE_DATA[0].uri,
+                    casedbUri = CASE_DATA[1].uri,
+                    editor = label.ckeditor().editor,
+                    widget = util.getWidget('itext-en-label');
+                widget.input.promise.then(function () { 
+                    editor.on('change', function() {
+                        assert.equal(mug.p.labelItext.get(), 'question1<output value="#case/child/dob" />');
+                        assert.equal(getInstanceId(mug.form, sessionUri), "commcaresession");
+                        assert.equal(getInstanceId(mug.form, casedbUri), "casedb");
+                        util.assertXmlEqual(call("createXML"), CHILD_REF_OUTPUT_VALUE_XML,
+                                            {normalize_xmlns: true});
+                        done();
+                    });
+                    assert.equal(getInstanceId(mug.form, sessionUri), null);
+                    assert.equal(getInstanceId(mug.form, casedbUri), null);
+                    assert.equal(label.length, 1);
+                    util.findNode(dataTree, "dob").data.handleDrop(label);
+                });
+            });
+
+            describe("with two languages", function () {
+                beforeEach(function (done) {
```

Comment:
not sure. it's possible I was using it to reduce some boilerplate in some tests and just got into the habit of using it. I quickly went through and changed all of the tests that don't require beforeEach https://github.com/dimagi/Vellum/pull/652/commits/a37daef0f0aadef713d6da634515b45fd1ec96b0

Other tests depend on it in some way (though some of them look easy to change to use util.loadxml("");)